### PR TITLE
setquota: add page

### PR DESCRIPTION
**Pull request by manny-uncharted**
fixed issue: #9342 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/setquota.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota.8>.
```
Comment (navarroaxel) [inline pages/linux/setquota.md:4]:
```suggestion
> More information: <https://manned.org/setquota>.
```

File: pages/linux/setquota.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
+
+- Set user quotas for named user:
+
+`setquota -u, --user`
```
Comment (navarroaxel) [inline pages/linux/setquota.md:12]: We can't create tldr pages only mention an isolated argument. The tldr pages are built with full working examples like the following:
```suggestion
`setquota --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
```

File: pages/linux/setquota.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota.8>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
+
+- Set user quotas for named user:
```
Comment (navarroaxel) [inline pages/linux/setquota.md:10]:
```suggestion
- Set a block and inode quotas for a specific user in a mounted filesystem:
```

File: pages/linux/setquota.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# setquota
```
Comment (navarroaxel) [inline pages/linux/setquota.md:1]: If the command it's available in more than 1 platform it should be placed on the `common/` directory. Please move the file.

**Comment by manny-uncharted** [comment]
Okay I'll effect that now

File: pages/linux/setquota.md (hunk 2 above)
Comment (manny-uncharted) [reply pages/linux/setquota.md]: Would resolve this now

File: pages/common/setquota.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
+
+- Set user quotas for named user:
+
+`setquota -u, --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
```
Comment (navarroaxel) [inline pages/common/setquota.md:12]:
```suggestion
`setquota --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
```

File: pages/common/setquota.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
```
Comment (navarroaxel) [inline pages/common/setquota.md:8]: Could you try to fix the other examples?

File: pages/common/setquota.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`setquota -u, --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
+
+- Set group quotas for named group:
+
+`setquota -g, --group`
+
+- Use quota settings of user, group or project to set the quota for the named user, group or project:
+
+`setquota -p, --prototype={{protoname}}`
```
Comment (kbdharun) [inline pages/common/setquota.md:20]:
```suggestion
`setquota --prototype={{protoname}}`
```

File: pages/common/setquota.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+`setquota -r, --remote`
+
+- Set user quotas for named user:
+
+`setquota -u, --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
+
+- Set group quotas for named group:
+
+`setquota -g, --group`
```
Comment (kbdharun) [inline pages/common/setquota.md:16]:
```suggestion
`setquota --group`
```

File: pages/common/setquota.md (hunk 6 above)
Comment (kbdharun) [inline pages/common/setquota.md:8]:
```suggestion
`setquota  --remote`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page. As @navarroaxel suggested above we prefer longer flags.

File: pages/common/setquota.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota  --remote`
+
+- Set user quotas for named user:
```
Comment (EmilySeville7cfg) [inline pages/common/setquota.md:10]:
```suggestion
- Set user quotas for a specific user:
```

Optional fix: `a specific` may sound better.

File: pages/common/setquota.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+`setquota --user {{username}} {{block_soft_limit}} {{block_hard_limit}} {{inode_soft_limit}} {{inode_hard_limit}} {{path/to/directory}}`
+
+- Set group quotas for named group:
```
Comment (EmilySeville7cfg) [inline pages/common/setquota.md:14]:
```suggestion
- Set group quotas for a specific group:
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory unmentioned fixes found, approved.
- This page should be reviewed by `setquota` users.
- Thanks for a contribution! 😄

File: pages/common/setquota.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`setquota --group`
```
Comment (EmilySeville7cfg) [inline pages/common/setquota.md:16]:
```suggestion
`setquota --group {{value}}`
```

Optional fix: invalid command syntax: argument for `--group` is expected.
Note: addresses [this](https://github.com/tldr-pages/tldr/pull/9413/files#r1019373423) comment.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
According to our maintainers guide we can apply suggestions ourselves now.

**Pull request closed without merging**